Classes/Eel/ElasticSearchQueryBuilder.php

```diff
@@ -131,6 +138,97 @@ public function nodeType(string $nodeType): QueryBuilderInterface
         return $this->queryFilter('term', ['neos_type_and_supertypes' => $nodeType]);
     }
 
+    /**
+     * Filter multiple node types
+     *
+     * @param string|array $nodeTypeFilter
+     * @return ElasticSearchQueryBuilder
+     * @throws QueryBuildingException
+     * @api
+     */
+    public function nodeTypeFilter($nodeTypeFilter): QueryBuilderInterface
```

Comment:
These days I would rather not implement such a "messy" interface anymore, but instead have a signature like
`nodeTypeFilter(array $expectedNodeTypes, array $excludedNodetypes)` (or introduce a filter DTO but that then would probably have to reside in the CR rather than here).

Reply:
You're right two parameters are cleaner :)
I updated the PR.